README.md

```diff
@@ -4,3 +4,121 @@ A central place for introspecting and executing the various modular components o
 [![Tests](https://github.com/pangeo-forge/pangeo-forge-orchestrator/actions/workflows/main.yaml/badge.svg)](https://github.com/pangeo-forge/pangeo-forge-orchestrator/actions/workflows/main.yaml)
 [![pre-commit](https://github.com/pangeo-forge/pangeo-forge-orchestrator/actions/workflows/pre-commit.yaml/badge.svg)](https://github.com/pangeo-forge/pangeo-forge-orchestrator/actions/workflows/pre-commit.yaml)
 [![Documentation Status](https://readthedocs.org/projects/pangeo-forge-orchestrator/badge/?version=latest)](https://pangeo-forge-orchestrator.readthedocs.io/en/latest/?badge=latest)
+
+
+## Database Migrations with Alembic
+
+This application is configured to use Alembic database migrations.
+The configuration was pieced together from the following resources:
+
+- Generic Alembic tutorial: https://alembic.sqlalchemy.org/en/latest/tutorial.html
+- Using sqlmodel with Alembic: https://github.com/tiangolo/sqlmodel/issues/85#issuecomment-917228849
+
+### Relevant Files
+
+- `alembic.ini` - Alembic configuration file
+- `migrations/env.py` - The migration environment; customized to work with
+  sqlmodel our test / deployment environment
+- `migrations/versions/*.py` - Auto-generated scripts which perform the migrations.
+
+### `DATABASE_URL` Environment Variable
+
+Both the migration environment and the FastAPI app itself are configured to use
+the `DATABASE_URL` environment variable to determine which database to connect to.
+**This environment variable must be set in order for the app to work.**
+This string must be a valid sqlalchemy database URI.
+
+For example, to set up a local environment for the test suite:
+
+```bash
+export DATABASE_URL=sqlite:///`pwd`/database.sqlite
+pytest -vx tests
+```
+
+The Heroku deployment environments automatically set this environment variable
+to point to the appropriate postgres database.
+
+### Running Migrations
+
+The command
+
+```bash
+python -m alembic upgrade head
+```
+
+will initialize the database located at `DATABASE_URL` to the latest version of
+the schema. If no tables are present, it will create them. If a different version
+exists, it will migrate. For more details on how this works, see the Alembic docs.
+
+**A migration must be run before the app is started on a fresh database!**
+Otherwise the tables will not exist. (The app code itself no longer creates tables.)
+
```

Comment:
```suggestion

> ### Local Testing 
> 
> What we've learned in the previous two sections about the `DATABASE_URL` variable and the requirement that migrations be run prior to starting the API informs the local testing approach. To set up local testing, run
>
> ```bash
> export DATABASE_URL=sqlite:///`pwd`/database.sqlite
> ```
>
> to assign the required environment variable. Then run the migrations with
> 
> ```bash
> python -m alembic upgrade head
> ```
> 
> And finally, invoke the tests
>
> ```bash
> pytest -vx
> ```
>
> Note that `.sqlite` files are excluded by `.gitignore`, so you don't need to worry about the database file appearing in your commit.
```

Reply:
I thought it might be nice to inset this as a blockquote, since it's a bit of an aside to the main narrative here. But not attached to that stylistic choice.